docs/iam.rst

```diff
@@ -390,7 +390,7 @@ In case you are using SGE, Slurm or Torque as a scheduler:
                   "iam:SimulatePrincipalPolicy"
               ],
               "Effect": "Allow",
-              "Resource": "arn:aws:iam::<AWS ACCOUNT ID>:role/<PARALLELCLUSTER EC2 ROLE NAME>"
+              "Resource": "arn:aws:iam::<AWS ACCOUNT ID>:role/parallelcluster-*"
```

Comment:
what if the EC2 role name is created by the customer and doesn't start with parallelcluster-?

Reply:
This is only the customer's role if they're using `ec2_iam_role`. Perhaps a note is best. The current wording `<PARALLELCLUSTER EC2 ROLE NAME>` leads one to believe it's the user role (aka the role this policy is attached too) when in fact it's the root role created by the cloudformation template.

Reply:
We also use `parallelcluster-*` in the batch version of this policy. See [#L561](https://github.com/aws/aws-parallelcluster/blame/develop/docs/iam.rst#L561)

Reply:
I've updated this with a comment

Reply:
Yeah i was referring exactly to the case when the user is using a custom ec2 role. I think the comment is good enough